usecases/schema/validation.go

```diff
@@ -97,13 +101,34 @@ func validateNestedPropertyTokenization(property *models.NestedProperty,
 		switch primitiveDataType {
 		case schema.DataTypeText, schema.DataTypeTextArray:
 			switch property.Tokenization {
```

Comment:
(GH doesn't let me do a normal "suggestion" here) We probably could group the `property.Tokenization == ""` case in the first switch-case:

```go
switch property.Tokenization {
    case models.PropertyTokenizationField, models.PropertyTokenizationWord, ..., "":
        return nil
    // other cases ...
    default:
        return fmt.Errorf(...)
}
```

Then we could also remove the `if property.Tokenization == "" { return nil }` block on line 139 ✨